hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -626,7 +626,7 @@ points for third-parties to integrate their authentication and authorization
 services into the ABFS client.
 
 * `CustomDelegationTokenManager` : adds ability to issue Hadoop Delegation Tokens.
-* `AbfsAuthorizer` permits client-side authorization of file operations.
+* `SASTokenProvider` permits client-side authorization of file operations.
```

Comment:
Client-side makes it sound like the client authorizes itself, which is not the case.  I suggest " `SASTokenProvider` allows for custom provision of Azure Storage Shared Access Signature (SAS) tokens." to keep with the pattern on line 630.

Reply:
Done